MonoGame.Framework/Platform/SDL/SDLGameWindow.cs

```diff
@@ -299,10 +299,17 @@ public void ClientResize(int width, int height)
                 _game.GraphicsDevice.PresentationParameters.BackBufferHeight == height) {
                 return;
             }
+
+            Viewport prevViewport = _game.GraphicsDevice.Viewport;
+            Rectangle prevScissorRect = _game.GraphicsDevice.ScissorRectangle;
+
             _game.GraphicsDevice.PresentationParameters.BackBufferWidth = width;
             _game.GraphicsDevice.PresentationParameters.BackBufferHeight = height;
             _game.GraphicsDevice.Viewport = new Viewport(0, 0, width, height);
 
+            if (_game.GraphicsDevice.RasterizerState.ScissorTestEnable && prevScissorRect == prevViewport.Bounds)
```

Comment:
A question, if the check is only on the PREVIOUS viewport bounds (and not the updated version in lines 306-308), then why is this change AFTER the viewport update?

1. If the Scissor check is based on the PREVIOUS viewport bounds, then this condition should be moved to line 305.
2. If the Scissor check is AFTER the update to the viewport bounds, should this not be validating against the NEW and not the previous bounds?

Reply:
After looking over it, it should have been at line 305 as you stated. Just pushed an update.